### PR TITLE
tests(psycopg2): add encoding test

### PR DESCRIPTION
**Pull request by majorgreys**
## Description

Added snapshot test for psycopg for encoding tests.

### CODE REVIEW

File: riotfile.py [hunk 1]
```diff
@@ -203,6 +203,30 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                 ),
             ],
         ),
+        Venv(
```
Comment (brettlangdon) [inline riotfile.py:206]: is there a corresponding change to `tox.ini` needed?

**Review by brettlangdon — COMMENTED** [review]
it'll be better to break up these changes in two PRs, but 👍🏻 let's do them both

File: riotfile.py (hunk 1 above)
Comment (majorgreys) [reply riotfile.py:206]: Missed that in the initial commit, updated!

File: tox.ini [hunk 2]
```diff
@@ -91,10 +91,6 @@ envlist =
     mysql_contrib-py{27,35,36,37,38,39}-mysqlconnector{80,}
     mysqldb_contrib-py27-mysqldb{12,}
     mysqldb_contrib-py{27,35,36,37,38,39}-mysqlclient{13,14,}
-    psycopg_contrib-py{27,35,36}-psycopg2{24,25,26,27,28}
-    psycopg_contrib-py37-psycopg2{27,28}
-# psycopg <2.7 doesn't support Python 3.8: https://github.com/psycopg/psycopg2/issues/854
-    psycopg_contrib-py{38,39}-psycopg2{28}
     pylibmc_contrib-py{27,35,36,37,38,39}-pylibmc{140,150,}
     pylons_contrib-py27-pylons{096,097,010,10,}
     pymemcache_contrib{,_autopatch}-{py27,py35,py36,py37,py38}-pymemcache{130,140}
```
Comment (jd) [inline tox.ini:96]: You should also remove the dependencies list like `pyscopg224` etc below.

File: tests/contrib/psycopg/test_psycopg.py [hunk 3]
```diff
@@ -291,6 +292,25 @@ def test_composed_query(self):
             dict(name='postgres.query', resource=query.as_string(db)),
         )
 
+    @snapshot()
```
Comment (jd) [inline tests/contrib/psycopg/test_psycopg.py:295]: would have been great to have 2 different changes for this though, one for riot and one after for this test.

**Review by jd — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 2 above)
Comment (majorgreys) [reply tox.ini:96]: Unfortunately that requires moving sqlalchemy over as well. Will do that in a separate PR.

File: tests/contrib/psycopg/test_psycopg.py (hunk 3 above)
Comment (majorgreys) [reply tests/contrib/psycopg/test_psycopg.py:295]: Moved this into #2013 cc @brettlangdon

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**